app/data/projects.json

```diff
@@ -2798,5 +2798,14 @@
     "category": "Operations",
 		"twitterHandle": "",
 		"twitterURL": ""
+  },
+  {
+    "projectName": "Copr",
+    "projectDescription": "A service that builds your open-source projects and creates your own RPM repositories"
+    "projectRepository": "https://github.com/fedora-copr/copr",
```

Comment:
Shouldn't this point to the upstream location (https://pagure.io/copr/copr)?

Reply:
You are right @Conan-Kudo,
thank you.

When I saw the page title "RED HAT ON GITHUB", I thought the `projectRepository` must point to GitHub. But there are some other projects hosted on pagure ...

Updated, PTAL